### PR TITLE
ZVISION: Boost volume for MPEG cutscenes

### PR DESCRIPTION
**Pull request by eriktorbjorn**
The high-resolution videos play back at much lower volume than the original ones. This adds hard-coded values for how much to amplify each cutscene. It's all done by ear, and it does introduce some
clipping, but I think it should be acceptable. 

Of course, it could also be a problem with the audio decoder, so this may be the wrong approach entirely.

### CODE REVIEW

**Comment by eriktorbjorn** [comment]
I forgot to mention that this is in response to my own bug report #10862, "AUDIO: AC3 volume seems too low".

**Comment by eriktorbjorn** [comment]
> What are your thoughts on variable change from decibel to gain_dB?

I don't have any strong opinions for or against. At least not at the moment.

File: engines/zvision/video/video.cpp [hunk 1]
```diff
@@ -49,8 +49,99 @@ Video::VideoDecoder *ZVision::loadAnimation(const Common::String &fileName) {
...
+		} else if (tmpFileName == "g000d981.vob") {
+			// The Enterprise logo. Has no low-res version. Its
+			// volume is louder than the other logo animations.
+			amplification = 6.2;
+		} else if (tmpFileName.hasPrefix("g000d")) {
+			// The Dolby Digital and Activision logos. They have no
+			// low-res versions, but I've used the low-resolution
+			// Activision logo (slightly different) as reference.
+			amplification = 8.5;
+		}
```
Comment (bluegr) [inline engines/zvision/video/video.cpp]: IMHO, this code block that decides the amplification for each video should be moved into a separate function

**Comment by bluegr** [comment]
Well done! I believe this is good for now, since no other engine is using the AC3 decoder

**Pull request merged**